livekit_ingress.proto

```diff
@@ -0,0 +1,90 @@
+syntax = "proto3";
+
+package livekit;
+
+import "google/protobuf/empty.proto";
+import "livekit_models.proto";
+
+option go_package = "github.com/livekit/protocol/livekit";
+
+service Ingress {
+    rpc CreateIngress(CreateIngressRequest) returns (IngressInfo);
+    rpc ListIngress(ListIngressRequest) returns (ListIngressResponse);
+    rpc DeleteIngress(DeleteIngressRequest) returns (google.protobuf.Empty);
+}
+
+enum IngressInput {
+    RTMP_INPUT = 0;
+//    FILE_INPUT = 1;
```

Comment:
support in future?

Reply:
yeah, will uncomment them when we start implementing them.